src/reconcile/rec_write.c

```diff
@@ -2462,6 +2462,7 @@ __rec_split_write(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REC_CHUNK *chunk
         /* We have an empty page. Free the multi. */
         if (chunk->entries == 0 && !multi->supd_restore) {
             WT_ASSERT(session, F_ISSET(btree, WT_BTREE_DISAGGREGATED));
+            __wt_rec_set_updates_durable(btree, multi);
```

Comment:
There is another issue here that we are not marking the updates before we free the multi.